swagger-ui/src/main/resources/index.html

```diff
@@ -1,142 +0,0 @@
-<!--
```

Comment:
Yes it would be exposes to end user. As much as I'd love to use "it's a security issue" as an excuse not to add it, by its nature as a purely HTML/JS frontend that's not an argument that holds.